### DISCUSSION TITLE
Works great on Windows 11 (installing from source)

### DISCUSSION THREAD

**Discussion by DRSDavidSoft** [pystardust/ani-cli#1357] [General] [answered]
Hi there, I've always wanted to install `ani-cli` and I just noticed I already have all of the requirements on my machine (other than `fzf` which I promptly installed). No need for scoop! No WSL! (I use FFmpeg at work, and use mpv/YT-DLP for general video/Youtube, ettc.)

I cloned the repo to `AppData` directory and wrote a small file to call it using `sh.exe` instead:

```
C:\Users\David\AppData\Roaming\ani-cli (master -> origin)
λ cat ani-cli.cmd

@SET SHLVL=2
@sh -l "%~dp0/ani-cli" %*
@SET SHLVL=
```

This works **great** with [**Cmder**](https://github.com/cmderdev/cmder), my favorite shell. Since it uses `cmd.exe` under the hood with [`clink`](https://github.com/chrisant996/clink), the script above calls `sh.exe` and runs `ani-cli` in it. Also, Windows Terminal for the win! 👍🏻 

One thing of note is the `--login` or `-l` flag which also makes the `git-for-windows` be prepended to PATH. This is required to make commands like `sort` and `find` from git to be preferred over their Windows counterparts which breaks ani-cli.

### Demo

https://github.com/pystardust/ani-cli/assets/4673812/532ab7e7-1f7e-4bce-9146-1bec2c28c1f9

**Comment by ykhan21** [comment, marked as answer]
@DRSDavidSoft, thanks for the info.

Scoop does cause a lot of problems now for ani-cli. scoop installs a shim which used to call git's bash but now calls wsl's bash. This shim is triggered in powershell or cmd. A normal windows installation of ani-cli isn't supposed to use wsl.

scoop for other than ani-cli works well to install and update dependencies: a proper terminal (windows terminal, cmder, wezterm, etc), git (which cmder apparently comes with), mpv, and fzf.

There is a PR to fix the scoop shim (https://github.com/ScoopInstaller/Extras/pull/13342), but if that doesn't go through, we might want to remove ani-cli from scoop and suggest a manual installation.

**Comment by DRSDavidSoft** [comment]
Great info! I'm sad to say I never liked scoop personally, it doesn't give me the feeling of the great package managers such as `pacman`, `yay`, `apt`, `dnf` that I use on other systems.

I don't believe it'd be best to remove ani-cli from scoop for those people who like to use scoop though, so I'm hoping the mentioned PR goes through! 👍🏻 

It doesn't make sense to use WSL's bash over native-provided ones. (P.S. As a long time Cygwin user, it's so sad that Microsoft discontinued WSL/1 which was closer to the way mingw and git for windows work, but that's just me 😄)

**Comment by ykhan21** [comment]
Also, scoop might decide to change the generated shims, breaking it again.

Someone could add ani-cli to Windows msys, which has Arch’s pacman. The msys pkgbuild could be derived from Arch pacman's ani-cli pkgbuild. The msys environment is pretty isolated and shouldn't get messed up by wsl.
https://www.msys2.org/dev/new-package/
________________________________
